cfme/tests/cloud/test_targeted_refresh.py

```diff
@@ -20,9 +52,40 @@ def test_ec2_targeted_refresh_instance():
             2. Instance RUNNING
             3. Instance STOPPED
             4. Instance UPDATE
-            5. Instance DELETE - or - Instance TERMINATE
+            5. Instance RUNNING
+            6. Instance DELETE - or - Instance TERMINATE
     """
-    pass
+    vms_collection = appliance.rest_api.collections.vms
+    flavors_collection = appliance.rest_api.collections.flavors
+
+    # create
+    template_id = provider.mgmt.get_template(
+        provider.data.templates.get('small_template').name).uuid
+    instance = provider.mgmt.create_vm(template_id, vm_name=random_vm_name('refr'))
+    if not instance:
+        pytest.fail("Instance wasn't successfully created using API!")
```

Comment:
This code shouldn't be necessary.

Reply:
Any thoughts here?

Reply:
Deleted as it is no longer needed when it's handled by create_vm.